### PR TITLE
admin page components

### PR DESCRIPTION
**Pull request by deanmarquesjr**
Admin page components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,16 +4,45 @@ import "bootstrap/dist/css/bootstrap.min.css";
...
+
+
 
 
 function App() {
   return (
     <div className="App">
       <Header />
-      <PackageCard />
+      {/* <PackageCard /> */}
```
Comment (Shane59) [inline src/App.js:24]: I believe you did not modify here

File: src/App.js [hunk 2]
```diff
@@ -4,16 +4,45 @@ import "bootstrap/dist/css/bootstrap.min.css";
...
       <Routes>
+        <Route path='/' element={<PackageCard
```
Comment (Shane59) [inline src/App.js:26]: Here as well

Comment (deanmarquesjr) [reply src/App.js:25]: I commit the App.js again, is that what you wanted fixed? I am a little confused on what you are asking

Comment (Shane59) [reply src/App.js:25]: I don't know why this part is shown as a modification even though this code seems to be in the main branch already. I just wanted to make sure your PR only includes the modifications you made

**Review by Shane59 — APPROVED** [review]

**Review by chk8089 — APPROVED** [review]

**Pull request merged**